### PR TITLE
Add issue template pointing people towards JIRA

### PR DESCRIPTION
**Pull request by meichstedt**
Github issues are still enabled for reference and to keep links alive. Instead of having people write a nicely formatted issue in markdown syntax and then letting them see it being closed, locked and migrated to JIRA, we should directly hint that they might consider creating an issue directly in JIRA.

We should consider adding a pull request template as well, and move github related files to a `.github` folder later. See https://github.com/blog/2111-issue-and-pull-request-templates for reference.

### CODE REVIEW

**Comment by janisz** [comment]
Github issues created by a bot are automatically assigned and has some labels. Is it important for the issue to have this labels and to be assigned?

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Head over to JIRA!
+
+Please note that we use JIRA for issue tracking and only keep GitHub issues around to not break existing links. Please prefer filing issues [directly in JIRA](https://jira.mesosphere.com/secure/CreateIssue!default.jspa?pid=10401).
+
+You can sign in to JIRA using github or google as identity providers.
```
Comment (ichernetsky) [inline ISSUE_TEMPLATE.md]: `Github` and `Google`?

Comment (meichstedt) [reply ISSUE_TEMPLATE.md]: Will do ;)

**Comment by meichstedt** [comment]
@janisz Actually, no. The labels were used to find issues created in Github (a.k.a by the community), but we try to process issues regardless of their origin now … which seems politically more correct, and also more helpful for us. We can differentiate in JIRA wrt the permissions of the user (Mesosphere employee vs the rest of the world) anyways.

I recently changed the labels to be `github` only, but that hasn't been updated yet.

**Comment by unterstein** [comment]
@meichstedt I would add that logs are always highly appreciated :)

LGTM

**Comment by jeschkies** [comment]
@meichstedt, can we merge this?

**Pull request merged**